src/utils/index.ts

```diff
@@ -84,12 +88,24 @@ export const toFixed = function (x) {
 export const trimCurrencyForWhales = (labelValue) => {
   // Nine Zeroes for Billions
   return Math.abs(Number(labelValue)) >= 1.0e9
-    ? (Math.abs(Number(labelValue)) / 1.0e9).toFixed(2) + 'B'
+    ? (Math.abs(Number(labelValue)) / 1.0e9).toFixed(
```

Comment:
thought process here?

Reply:
When the "Big BANK" number gets to be too many characters it goes exceeds the bounds of the box that it sits in on the current design.

Example if the number is 100k it is fine, but if it is 100.1k it is not. So the length is checked here to keep the final number accurate.

Reply:
This is what I mean by it clipping the bounds of the box. If you remove the decimal this number works fine. With the change to the code I made, nobody will ever see this happen:

![CleanShot 2021-06-08 at 20 54 41](https://user-images.githubusercontent.com/81541283/121280350-c83f2680-c89b-11eb-9895-74e2c4ec69c9.png)

Reply:
I don't like the decimal anyway; this just removes it? Alternatively, just reduce font size lol

Reply:
It removes the decimal when the length of the number to the left of the decimal is 3 characters or more.

Examples

  **a.**  100 = 3 characters, so `toFixed(0)`
  **b.**  99.12 = 2 characters left of decimal, so `toFixed(1)` keeps it at 99.1 
  **c.**  9.13 = 1 character left of decimal, so `toFixed(1)`* keeps it as 9.1

**side note this could be `toFixed(2)` but whatevs*

If you want to drop the decimal in all cases to simplify this code that is fine. I don't mind doing that for the sake of clarity but also I kinda like the way it looks when we can show/hide the decimal in cases **a** and **b** which is what most people see. 

Changing the font size so it fits seems like it would throw off the padding a bit so i'm less inclined to do that.

Reply:
@wpmonty I've ultimately decided to refactor the method to simplify it. 

The `trimCurrencyForWhales` method now has as an optional parameter for setting the nearest decimal when rounding. 
```
trimCurrencyForWhales = (
  labelValue: number,
  nearest = 2
): string | number =>
```
This is used on make `BigBank` use `toFixed(1)` on the large $BANK balance. 

At the same time I implemented your suggestion to make the font slightly smaller on mobile to compensate for scenario I showed in [the screenshot above](https://user-images.githubusercontent.com/81541283/121280350-c83f2680-c89b-11eb-9895-74e2c4ec69c9.png). It looks fine even with smaller numbers so i'm happy with that.

I believe this is a cleaner implementation overall. It also helped clean up a small typescript warning on the `trimCurrencyForWhales` function.